(DOCS-2753) Remove instances of 'easy' and 'easily' (#12469)
DataDog/documentation: content/en/monitors/faq/i-have-a-downtime-scheduled-on-my-monitor-why-did-it-still-alert.md

If you want to schedule a downtime over a subset of hosts, a good way to do so is to find a host-tag common to all those hosts and scope the downtime by that tag.

> One way you can take advantage of this is to create new host-tags directly in the Datadog UI from your [Infrastructure List][2] and [Host Map][3] by selecting a host and hitting the **Edit Tags** button (shown below), or via [our API][4].

Reviewer: ```suggestion
One way you can take advantage of this is to create new host-tags directly in the Datadog UI from your [Infrastructure List][2] and [Host Map][3] by selecting a host and hitting the **Edit Tags** button (shown below), or with the [Datadog API][4].
```
